spec/tests/AdTracking.spec.ts

```diff
@@ -0,0 +1,463 @@
+import { AdTrackingMode, ConvivaAnalytics, ConvivaAnalyticsConfiguration } from '../../src/ts/ConvivaAnalytics';
```

Comment:
why are we using a different test name pattern?

Reply:
Different to which one?

Reply:
`AdTrackingSpec.ts`, the one we use in the player. But I guess this is fine.

Reply:
Ah I thought you mean within the project. Anyway I just used the `jest` default pattern.